js/pokemap.js

```diff
@@ -4,37 +4,39 @@ var util = require('util');
 var L = require('leaflet');
 var io = require('socket.io-client');
 var socket = io.connect("http://localhost:3000");
-var Pokemon = require('./basictypes');
 var config = require('./config');
 require('leaflet.locatecontrol');
+require('leaflet-routing-machine');
+require('leaflet-control-geocoder');
 
 var mymap = null;
-var apiURL={};
+var apiEndpoint=null;
 var getAllSightingsURL = "/api/pokemon/sighting";
 var getAllSightingsByTimeRangeURL = "/api/pokemon/sighting/ts/";
 var getAllPokemon = "/api/pokemon";
 var getPokemonById = "/api/pokemon/id/";
 var getAllPredictions = {};
 
-var PokeMap = function(htmlElement, options={filter : {pokemonIds: 0, sightingsSince: 0, predictionsUntil: 0}, tileLayer: config.currentMap,apiEndPoint : 'http://pokedata.c4e3f8c7.svc.dockerapp.io:65014'}) {
+var PokeMap = function(htmlElement, options={filter : {pokemonIds: 0, sightingsSince: 0, predictionsUntil: 0}, tileLayer: config.currentMap,apiEndpoint : 'http://pokedata.c4e3f8c7.svc.dockerapp.io:65014', websocketEndpoint: 'http://pokedata.c4e3f8c7.svc.dockerapp.io:65024'}) {
   this.htmlElement = htmlElement;
-  apiURL = options.apiEndPoint;
-
+  apiEndpoint = options.apiEndpoint;
+  websocketEndpoint = options.websocketEndpoint;
 
   // which pokemons should be shown; if null show all pokemons; otherwise only pokemons with ids in the list
   this.filterPokemons = null;
 
-  //this.socket = io.connect('http://localhost:3000');
+  socket = io.connect(websocketEndpoint + '/mobs');
```

Comment:
As far as I understand you only need to do `io.connect(websocketEndpoint)` here. The `/mobs` part will become important later when you do `socket.on('mobs', ..)`.